### DISCUSSION TITLE
ram usage?

### DISCUSSION THREAD

**Discussion by sedlund** [charmbracelet/mods#209] [Q&A]
im running mods on nix, and I notice after about the 3rd -C continuation of the same conversation mods us using like 1GB of ram on invocation.  In one really long convo it OOM'd my VM to swap death (6GB ram VM with 9GB of zram) where it killed it

```
Jan 30 13:46:00 fsa kernel: [  pid  ]   uid  tgid total_vm      rss pgtables_bytes swapents oom_score_adj name
Jan 30 13:46:00 fsa kernel: [3717276]  1000 3717276  3768783  1073135 26411008  2184467             0 mods
Jan 30 13:46:00 fsa kernel: oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=gitea-runner-codeberg.service,mems_allowed=0,global_oom,task_memcg=/user.slice/user-1000.slice/session-1.scope,task=mods,pid=3717276,uid=1000
Jan 30 13:46:00 fsa kernel: Out of memory: Killed process 3717276 (mods) total-vm:15075132kB, anon-rss:4292540kB, file-rss:0kB, shmem-rss:0kB, UID:1000 pgtables:25792kB oom_score_adj:0
Jan 30 13:46:00 fsa systemd-journald[538]: Under memory pressure, flushing caches.
Jan 30 13:44:30 fsa systemd-resolved[845]: Under memory pressure, flushing caches.
Jan 30 13:44:31 fsa systemd-resolved[845]: Flushed all caches.
```

is anyone else having memory problems with it?

also - the character scroller progress indicator thing is cute and all, but i dont know that I need that much data to go over the ssh wire on a cellular connection maybe put a delay in there?

> mods version 1.1.0

**Comment by bashbunni** [comment]
Hey @sedlund have you tried running this with the `-q` or `--quiet` flag? That would hide the spinner. I'm not sure what would cause that amount of memory usage off the cuff, but definitely let us know if hiding the spinner helps in your case. If you have any steps to reproduce, please pass them along. That would be really fun to test out on my own machine

**Reply by bashbunni** [reply]
Oh, just saw the issue as well. I'll close the discussion here so we keep the conversation going in one place. 
https://github.com/charmbracelet/mods/issues/211
